src/components/core/DpVideoPlayer.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpVideoPlayer from './DpVideoPlayer.vue'
+
+const meta: Meta<typeof DpVideoPlayer> = {
+    component: DpVideoPlayer,
+    title: 'Components/VideoPlayer',
+    render: (args) => ({
+        components: {
+            DpVideoPlayer
+        },
+        setup() {
+            return { args }
+        },
+        template: `<dp-video-player v-bind="args"></dp-video-player>`,
+    })
+}
+
+interface IDpVideoPlayer {
+    iconUrl: string
+    id: string
+    sources: object[]
+}
+
+type Story = StoryObj<IDpVideoPlayer>
+
+export default meta
+
+export const Default: Story = {
+    args: {
+        iconUrl: 'videoIconUrl.svg',
```

Comment:
If you provide an empty string, the plyr svg is actually found.

```suggestion
        iconUrl: '',
```

Maybe you could add this link to the docblock of the prop within DpVideoPlayer: https://github.com/sampotts/plyr#using-the-iconurl-option

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/590/commits/bb799bc52a78bff443895392a9d3f92762e43fd8